detekt/detekt.yml

```diff
@@ -19,7 +19,7 @@ formatting:
   FinalNewline:
     insertFinalNewLine: true
   MaximumLineLength:
-    maxLineLength: 100
+    maxLineLength: 120
```

Comment:
Tired of getting lint failures for this 😂

Reply:
lol 100 is really aggressive for a JVM lang